simulate.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+
+import numpy as np
+import scipy as sp
+from scipy import signal
+import h5py
+import math
+import PSS_utils as utils
+import sys
+sys.path.append('/Users/nearclouding/Desktop/UTGRV_REU/nanograv_group')
+import VersionZeroPointZero as PSS
+
+#Initializing the signal
+S1=PSS.signal(Nt=2000)
```

Comment:
Delete this. The idea is that someone will enter their own instance of `Signal()`, and then enter it as `signal_class` . One thing that might help you, is to think about writing a function for Numpy . All of our functions will be generic, until some else (or us) writes a script (whether in a notebook or on there own) to use the methods to simulate a signal.